src/DSharp.Mscorlib/Func.cs

```diff
@@ -4,37 +4,46 @@ namespace System
 {
     [ScriptIgnoreNamespace]
     [ScriptImport]
+    [ScriptIgnoreGenericArguments]
```

Comment:
we do not need to add ignore generics? if we choose to, we should also add the attribute to Action, List, Dictionary et. al.